cmd/controller/main.go

```diff
@@ -129,6 +129,11 @@ func main() {
 			knativeRuntime = true
 		}
 	}
+	if knativeRuntime {
+		logger.Info("Starting with Knative runtime")
+	} else {
+		logger.Warn("Starting without Knative runtime")
```

Comment:
Why a warning? Should this instead be: "starting with core runtime"?

Reply:
The core runtime is always installed.

It's a warning to make it more distinguished in the logs